### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by wpbonelli**
Sample test runs: https://github.com/w-bonelli/install-intelfortran-action/actions/runs/3022480580

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
...
+  uses: modflowpy/install-intelfortran-action@v0.0.1
+```
+
+Environment variables must be set before the compiler can be used. On Linux or MacOS runners, run `source /opt/intel/oneapi/setvars.sh`. On Windows, run `call "scripts/install/build_windows.bat"` from a step with `shell: cmd`.
+
+**Note:** Environment variables are not preserved between steps, so the above must occur in the same step in which the `ifort` command is used.
+
+## Disclaimer
+
+This software is preliminary or provisional and is subject to revision. It is being provided to meet the need for timely best science. The software has not received final approval by the U.S. Geological Survey (USGS). No warranty, expressed or implied, is made by the USGS or the U.S. Government as to the functionality of the software and related material nor shall the fact of release constitute any such warranty. The software is provided on the condition that neither the USGS nor the U.S. Government shall be held liable for any damages resulting from the authorized or unauthorized use of the software.
```
Comment (christianlangevin) [inline README.md]: I'd suggest that we do not use the usgs disclaimer here, as we probably shouldn't claim this as a usgs product.

**Review by christianlangevin — APPROVED** [review]
Looks good to me, only suggestion might be to add a hello world fortran program and then compile it and verify that it runs as part of a test.  This could be done later.

**Comment by wpbonelli** [comment]
@langevin-usgs added a test step to compile and run hello world and check its output

https://github.com/w-bonelli/install-intelfortran-action/actions/runs/3022480580

**Review by christianlangevin — APPROVED** [review]
I can't see any GH Action output on my end, but I assume it's working as intended.  Looks really nice, @w-bonelli.

**Pull request merged**